### DISCUSSION TITLE
Getting error while connecting to frontend

### DISCUSSION THREAD

**Discussion by Kavitha328** [learndevopsonline/learndevopsonline#883] [Q&A] [answered]
Hi Raghu,
Using Jenkins I was able to run terraform code for roboshop successfully:
<img width="799" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/125950657/f7ab35d1-cbc3-4206-8308-e60c72b6357d">

But while connecting to frontend using ansible getting below error.
<img width="945" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/125950657/30104941-eec5-4c52-996b-548b5e81cbe8">

IPs:
Workstation:35.172.209.150
Jenkins:44.214.93.197

**Comment by PranathiAT** [comment, marked as answer]
give -e before role_name=frontend.

**Reply by Kavitha328** [reply]
Got it now.Thanks
